content/docs/alerting/configuration.md

```diff
@@ -376,7 +376,7 @@ channel: <tmpl_string>
 ```
 
 
-## OpsGenie receiver `<opsgenie_config>`
```

Comment:
This defines one instance of an OpsGenie config. Hence, the missing 's' is intentional.